riscv_isac/coverage.py

```diff
@@ -276,7 +276,7 @@ class archState:
     Defines the architectural state of the RISC-V device.
     '''
 
-    def __init__ (self, xlen, flen):
+    def __init__ (self, xlen, flen, inxFlg):
```

Comment:
The `inxFlg` should not be passed to the functions in the `coverage.py`. To isolate such changes, the latest master has been refractored to localise all variable derivations to the instruction object. The changes in the `coverage.py` will have to be relocated into `instruction_object.py`

Reply:
The spec also says that when the `iflen < flen/xlen` the resulting value stored in the register should be sign extended, which means that these changes will not allow the tests for `RV64Zfinx` to get full coverage. Either the `define_sem` function should be updated to appropriately capture this. Infact, it can be treated similar to the `nan_prefix` variable where in case of the Zx extensions the expected prefix is the sign bit repeated `flen-iflen` times. @allenjbaum @neelgala, what do you think?